### ISSUE TITLE
Sysdig Agent configuration update for eBPF mode

### ISSUE THREAD

**Issue by jefkel** [BCDevOps/developer-experience#1101]
**Describe the issue**
eBPF mode supports a more secure approach for kernel monitoring than a full kernel module.  Sysdig has supported this technology for more than a year, and will be required for use with OpenShift 4.6.

**Additional context**
If this mode is validated and installed before the upgrade, then there will be no disruption to the sysdig metrics gathering post upgrade to 4.6.

**Definition of done**
Sysdig Agents configured to use eBPF instead of the legacy Kernel Module for gathering system metrics.  

SYSDIG_BPF_PROBE environment variable is defined:
- [ ] ARO.pathfinder
- [x] CLAB
- [x] KLAB
- [x] Silver
- [ ] Gold

**Comment by StevenBarre** [comment]
Blocked on the 2021Q2 Silver upgrade CHG

**Comment by StevenBarre** [comment]
Completed by CHG0025038 - MCS SILVER - 2021 Q2 Patching - Upgrade to OCP 4.6.21

**Comment by mitovskaol** [comment]
ARO.pathfinder is out of scope as the cluster is getting decommissioned.

**Comment by StevenBarre** [comment]
Gold will be completed with its upgrade in two weeks, no need to keep the ticket open
